Essentials/src/com/earth2me/essentials/commands/Commandeco.java

```diff
@@ -20,7 +20,7 @@
 public class Commandeco extends EssentialsLoopCommand {
     Commandeco.EcoCommands cmd;
     BigDecimal amount;
-
+	boolean isPercent;
```

Comment:
This should be 4 spaces to remain consistent with the code formatting in this file.

Reply:
Gotcha